docs/Installation Guide.md

```diff
@@ -236,6 +236,13 @@ In addition to this, you would also have to populate the database for django-cit
 
     python manage.py migrate
     python manage.py cities_light
+    
+if ImportError: No module named cities_light ,this means cities_light has not installed properly.
+For this perform the following steps:
+
+    pip uninstall django-cities-light
+    pip install django-cities-light==3.2.0
+    pip install -r requirements.txt
```

Comment:
@smarshy Yes I did. even I installed it 2 times because I was again and again facing the same problem. Also , this was faced by 2 more people . and when I told them to do these steps , they were able to rectify the problem . You can check the slack chats.